src/main/java/core/basesyntax/service/strategy/OperationStrategyImpl.java

```
@@ -0,0 +1,22 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.operation.OperationHandler;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private Map<Operation, OperationHandler> handlers;
+
+    public OperationStrategyImpl(Map<Operation, OperationHandler> handlers) {
+        this.handlers = handlers;
+    }
+
+    @Override
+    public OperationHandler get(Operation operation) {
+        OperationHandler handler = handlers.get(operation);
```

The `get(Operation operation)` method doesn't validate its input. If `operation` is null, handlers.get(operation) returns null and the method throws a RuntimeException with a message containing `null`, which is less explicit. Add a check `if (operation == null) { throw new RuntimeException("Operation must not be null"); }` to fail fast with a clearer message.